### PR TITLE
Adding a method of calling server methods from the command line

### LINKED ISSUE #71: run method from command line
There have been instances where I'd love to run a server method from the command line.  Here's what I'm thinking:

* rapptor checks if the first argument passed is `method`
* if it is, don't start the server, but still process everything (basically run `setup`)
* the next argument after `method` is the name of the method
* be able to support `namespaced.methods`
* be able to take in arguments like: `node server.js method jobs.fetchData test 123`
* be able to take in json arguments:  `node server.js method jobs.fetchData "{ test: 123 }"`
* the output of the method is displayed in the console via `JSON.stringify` to make it look nice

has anybody else needed something like this?

### PR DESCRIPTION
**Pull request by ecwillis**
Adds the ability for a server method to be called from the command line. 

Uses the syntax: `node index.js method [methodName] (arg1, arg2, arg3)` to call. Number of arguments must match the number of arguments for the method.

Note, arguments can be called as JSON strings as they will be parsed before hand. However, make sure it is __VALID__ JSON.

Fixes #71

### CODE REVIEW

File: lib/bootstrap.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+var argv = require('yargs').argv;
```
Comment (jgallen23) [inline lib/bootstrap.js]: I don't think we need yargs for this.  just do `process.argv`

File: index.js [hunk 2]
```diff
@@ -261,6 +264,40 @@ Rapptor.prototype.setup = function(callback) {
...
+  if(methodFunc) {
+    var funcArgs = argv._;
+
+    for(i=0; i<funcArgs.length; i++){
+      funcArgs[i] = JSON.parse(funcArgs[i]);
+    }
+    
+    funcArgs.push(function(err, data) {
+      if(err) {
+        console.log('An ERROR Occured'.red);
```
Comment (jgallen23) [inline index.js]: having `colors` change the string prototype worries me a bit.  I don't think it's worth it

File: index.js [hunk 3]
```diff
@@ -261,6 +264,40 @@ Rapptor.prototype.setup = function(callback) {
...
+        process.exit();
+      }
+
+      console.log(('Method '+method+' Result').yellow);
+      console.log(JSON.stringify(data, null, ' '));
+      process.exit();
+
+    });
+
+    methodFunc.apply(this, funcArgs);
```
Comment (jgallen23) [inline index.js]: I think `this` should be the server instead of the rapptor instance

File: index.js [hunk 4]
```diff
@@ -270,15 +307,24 @@ Rapptor.prototype.start = function(callback) {
       return callback(err);
     }
 
-    server.start(function(err) {
-      if (!err) {
-        server.log(['server', 'info'], 'Server started '+ server.info.uri);
-      }
+    var args = bootstrap.argv;
```
Comment (jgallen23) [inline index.js]: process.argv is built in and should work here instead of bootstrap

File: example/conf/default.yaml [hunk 5]
```diff
@@ -25,6 +25,8 @@ logging:
                 notify: true
 assets:
   host: 'http://localhost:8081'
+structure:
```
Comment (jgallen23) [inline example/conf/default.yaml]: what does this do?

Comment (ecwillis) [inline example/conf/default.yaml]: @jgallen23 The example was broken... it couldn't find the layout as it was configured so I added this to fix it.

File: index.js [hunk 6]
```diff
@@ -261,6 +261,40 @@ Rapptor.prototype.setup = function(callback) {
   ], callback);
 };
 
+Rapptor.prototype.callMethod = function(server, method, argv, callback) {
+
+  callback = callback || _.noop;
```
Comment (jgallen23) [inline index.js]: this isn't being used anywhere

**Pull request merged**